tests/phpunit/admin/migrations/test-class-wc-stripe-subscriptions-repairer-legacy-sepa-tokens.php

```diff
@@ -158,11 +158,17 @@ function ( $id ) {
 		$ids_to_migrate = $this->get_subs_ids_to_migrate();
 
 		$this->logger_mock
-			->expects( $this->at( 1 ) )
+			->expects( $this->exactly( 2 ) )
```

Comment:
Why do we need to update these tests? The new `update_main_stripe_settings` does not log anything 🤷🏼 
Do we know where is the additional log entry coming from?

Reply:
`at()` [was deprecated in PHPUnit 9](https://github.com/sebastianbergmann/phpunit/issues/4297). So, it threw many warnings when running the tests. I decided to refactor them here already. It is not a blocker, as we could wait for PHPUnit 10 (which removed this). I can also revert this if that is preferable.